Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_4_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Playbooks
+##### HAFNIUM - Exchange 0-day exploits
+- Expanded hunting coverage with XDR and Expanse
```

Comment:
Added more coverage to XDR and Expanse hunting
Added predefined Windows Event Logs hunting queries for SIEMs